components/scheduler/templates/slot.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Slot
+page_title: Scheduler - Slot Templates
+description: Use custom rendering for the slots in the Scheduler for Blazor.
+slug: scheduler-templates-dateheader
+tags: telerik,blazor,scheduler,templates,slot,alldayslot,dateheader
+published: True
+position: 15
+---
+
+# Slot Templates
+
+You can use the `SlotTemplate` and the `AllDaySlotTemplate` to customize the rendering of the slots in the Scheduler.
+
+* [AllDaySlotTemplate](#alldayslottemplate)
+* [SlotTemplate](#slottemplate)
+
+## AllDaySlotTemplate
+
+Use the `AllDaySlotTemplate` to provide a custom rendering for the cells in the Telerik Scheduler for Blazor that span across a full day. This template can be defined for the [day, multiday, and week Scheduler views]({%slug scheduler-views-overview%}). 
+
+The `context` of the template is a `SchedulerAllDaySlotTemplateContext` object that contains:
+
+| Property | Type | Description |
+| --- | --- | --- |
+| `Start` | `DateTime` | The slot's start time. |
+| `End` | `DateTime` | The slot's end time.|
+| `Resources` | `List<KeyValuePair<string, object>` | A collection of resources for which the slot is defined. The collection is populated when the [Resources]({%slug scheduler-resources%}) and [Resource Grouping]({%slug scheduler-resource-grouping%}) features are used. |
+
+## SlotTemplate
+
+Use the `SlotTemplate` to provide a custom rendering for the cells in the Telerik Scheduler for Blazor. This template can be defined for the [day, multiday, and week Scheduler views]({%slug scheduler-views-overview%}). 
```

Comment:
The SlotTemplate is available also for the Month and Timeline views of the Scheduler. The Month and Timeline views lack the AllDaySlotTemplate.

Reply:
thanks, will apply the change